docs/intelligentapps/playground.md

```diff
@@ -0,0 +1,38 @@
+---
+Order: 3
+Area: intelligentapps
+TOCTitle: Playground
+ContentId:
+PageTitle: AI Model Playground
+DateApproved:
+MetaDescription: Chat with selected generative AI model in playground. Change system prompt and parameters. Add attachment for Multi-Modal models. Keep chat history.
+MetaSocialImage:
+---
+
+# AI Toolkit playground
+
+The AI Toolkit playground is where you can interact with your AI models and try different prompts with different model parameter settings. You can also use the playground to interact with multi-modal models that support attachment of different input formats.
+
+In AI Toolkit, select the `Playground` in the treeview, or select `Load in Playground`/`Try in Playground` from a model card in the model catalog, you will enter playground view as screenshot:
+
+![Playground view](./images/playground/playground.png)
```

Comment:
Move this screenshot to the intro because it just provides an overview of the playground.